Lognormal tutorial

jeffjennings opened the pull request:
Adds a tutorial for lognormal fits. Closes #190

jeffjennings commented:
If there's any discussion you want to add to the tutorial, feel free.

rbooth200 commented:
I made some small changes to the tutorial, including correcting the visibilities for the non_negative model. You might want to re-run the tutorial yourself, the figrues etc look a little different when generated on my machine.

jeffjennings commented:
Great, thank you. I re-ran with a new seed (the fits vary a bit depending on the seed) and updated the text slightly. Also added a seed to the `mock_data` tutorial. Ready to merge when you approve.
